### ISSUE TITLE
ValueError: Invalid archive header

### ISSUE THREAD

**Issue by korshunovdv** [gmr/pgdumplib#8]
Hi, 
I try to use code from example:
```
import pgdumplib

dump_fname = "dump_26-05-2021_09_34_53.sql"
dump = pgdumplib.load(dump_fname)

print('Database: {}'.format(dump.toc.dbname))
print('Archive Timestamp: {}'.format(dump.toc.timestamp))
print('Server Version: {}'.format(dump.toc.server_version))
print('Dump Version: {}'.format(dump.toc.dump_version))

for line in dump.table_data('public', 'pgbench_accounts'):
    print(line)
```

But it's rise an error:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-94-8680c9bbd669> in <module>
      2 
      3 dump_fname = "dump_26-05-2021_09_34_53.sql"
----> 4 dump = pgdumplib.load(dump_fname)
      5 
      6 print('Database: {}'.format(dump.toc.dbname))

~/anaconda3/envs/data-engine/lib/python3.8/site-packages/pgdumplib/__init__.py in load(filepath, converter)
     22     from pgdumplib import dump
     23 
---> 24     return dump.Dump(converter=converter).load(filepath)
     25 
     26 

~/anaconda3/envs/data-engine/lib/python3.8/site-packages/pgdumplib/dump.py in load(self, path)
    232         self.entries = []  # Wipe out pre-existing entries
    233         self._handle = open(path, 'rb')
--> 234         self._read_header()
    235         if not constants.MIN_VER <= self.version <= constants.MAX_VER:
    236             raise ValueError(

~/anaconda3/envs/data-engine/lib/python3.8/site-packages/pgdumplib/dump.py in _read_header(self)
    550         """
    551         if self._handle.read(5) != constants.MAGIC:
--> 552             raise ValueError('Invalid archive header')
    553         self._vmaj = struct.unpack('B', self._handle.read(1))[0]
    554         self._vmin = struct.unpack('B', self._handle.read(1))[0]

ValueError: Invalid archive header
```

There is some issue on [Stack overflow](https://stackoverflow.com/questions/64586712/invalid-archive-header-at-open-a-dump-file), but still with no answer.

Can somebody help with this?

**Comment by Worschi** [comment]
Hi, 

I ran into the same problem. Im my case I tried a dump file that was create without the -Fc flag. 
So `pg_dump -d postgres -f localhost.dump` instead of correct `pg_dump -d postgres -Fc -f localhost.dump`

As far as I know, pgdumplib can only handle custom format dump files (a problem in my case, since I need to parse plain text files from 3rd party applications). Or did I just not find the plain text parser methods?
